### ISSUE TITLE
magisk install in secondary rom

### ISSUE THREAD

**Issue by gsarath19** [vasishath/multirom#13]
after install magisk seconday rom is going to fastboot

**Comment by forumi0721** [comment]
Me too,
Multirom is ok.
But recovery 07,05 had same problem.
Only recovery 03 works.

**Comment by madoolex** [comment]
Yes I had this issue after flashing magisk on secondary roms but i flash multirom zip on secondary rom  that worked for me on pie based roms
